ArkEcosystem/laravel-ui
# feat: add focus state styling

## Summary
https://app.clickup.com/t/rp4dfa

Since `focus-state` will be used across Ark applications, it makes sense to move Tailwind variants configuration and the rest of the CSS classes here so it can be easily shared.

It also adds `focus-state` to components like switch, info tooltip, footer, input validation tooltip, etc.

## How to test
Check open PRs:
- Nodem https://github.com/ArkEcosystem/nodem/pull/196
- Marketsquare https://github.com/ArkEcosystem/marketsquare.io/pull/2236

## Checklist
